crates/db_views_actor/src/community_view.rs

```diff
@@ -123,6 +123,8 @@ fn queries<'a>() -> Queries<
       let searcher = fuzzy_search(&search_term);
       let name_filter = community::name.ilike(searcher.clone());
       let title_filter = community::title.ilike(searcher.clone());
+      // TODO this currently filters the new short description column, but should it also check the
+      // sidebar too now?
```

Comment:
Not sure what should be done here, when they want to include the description filter. I could do either:

- Check all 4 (community name, title, description, and sidebar)
- Check 3 (name, title, description)

Reply:
Sidebar has a higher chance for false positives, as it includes all kinds of info about rules, related communities etc. So I would only check name, title and short description.

Reply:
K cool, its good as is then. I'll just remove the comment.